modules/imap/handler_modules.php

```diff
@@ -722,15 +722,23 @@ public function process() {
                 $this->session->set('imap_prefetched_ids', array_unique($prefetched, SORT_NUMERIC));
             }
             $with_subscription = isset($this->request->post['subscription_state']) && $this->request->post['subscription_state'];
+            $cache = Hm_IMAP_List::get_cache($this->cache, $form['imap_server_id']);
+            $imap = Hm_IMAP_List::connect($form['imap_server_id'], $cache);
+            if (imap_authed($imap)) {
+                $quota_root = $imap->get_quota_root('INBOX');
+                $quota = $imap->get_quota($quota_root[0]['name']);
+                if($quota && $quota[0]['max'] > 0) {
+                    $this->out('quota', ceil(($quota[0]['current'] / $quota[0]['max']) * 100));
```

Comment:
Please parse to int/float before dividing and check if max is not zero - both could throw errors in production.